preview/snap/snapcraft.yaml

```diff
@@ -84,10 +85,25 @@ parts:
       done >> $thirdPartyNoticeFile
       echo "Done building third party notices file - $thirdPartyNoticeFile"
     stage-packages:
-      - libicu55
+      - libicu60
```

Comment:
I assume this is the real fix here, compiling for the newest libicu version to avoid .NET Core picking the wrong one up from elsewhere?

Reply:
The problem was that .NET Core picks individual DLLs' newest to oldest, and it can see into the OS.  so on 18.04 it ends up picking a mix of old and new which don't work together.